language/e2e-tests/src/account_universe/peer_to_peer.rs

```diff
@@ -41,7 +41,10 @@ pub struct P2PNewReceiverGen {
 }
 
 impl AUTransactionGen for P2PTransferGen {
-    fn apply(&self, universe: &mut AccountUniverse) -> (SignedTransaction, TransactionStatus) {
+    fn apply(
+        &self,
+        universe: &mut AccountUniverse,
+    ) -> (SignedTransaction, (TransactionStatus, u64)) {
```

Comment:
Do we have a type alias for gas cost? If so we could use this here.

Reply:
@sunshowers we have `pub type GasCarrier = u64;` which according to @tzakian is the alias for what gets returned from a TransactionOutput.
Should we change all the u64 usage of gas cost to that?

Reply:
I am putting this in as is and look at this later, we'll talk and maybe I'll change all places where we use a u64